server/commands/index.js

```diff
@@ -184,13 +184,13 @@ export default class Commands {
     })
   }
 
-  claimPrrr(prrrId){
-    return this.queries.getPrrrById(prrrId)
+  claimPrrr(){
+    return this.queries.getNextPendingPrrr()
       .then(prrr =>
```

Comment:
what happens with `prrr` is null because there are no pending Prrrs? I think we need a test for this method. Please add one.